src/bin/nativelink.rs

```diff
@@ -72,9 +73,7 @@ use tokio_rustls::rustls::server::WebPkiClientVerifier;
 use tokio_rustls::rustls::{RootCertStore, ServerConfig as TlsServerConfig};
 use tonic::codec::CompressionEncoding;
 use tonic::service::Routes;
-#[cfg(target_family = "unix")]
```

Comment:
I'm not sure why this was ever here. Does `tracing::warn` not work on Windows or something?  That's such a particular invocation of `target_family` that I never noticed.